cli/src/main/scala/com/gu/localnews/cli/Main.scala

```diff
@@ -14,15 +14,14 @@ object Main extends App {
   implicit val ec = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(2))
 
   val client = ElasticsearchClient(List("http://127.0.0.1:9200"))
-  val index = new Index(client)
-
-  // Setup first time
-  index.setupCouncilContracts()
 
   val options = new Args(args)
 
   options.`type`() match {
     case ImportType.CouncilContracts =>
+      // Setup first time IS THIS RIGHT?
+      val index = new Index(client)
+      index.setupCouncilContracts()
```

Comment:
We should only need to do this once at boot time (i.e. where it was) unless I'm misunderstanding!

Reply:
@itsibitzi what's the correct way to do this?